src/ducks/balance/components/AccountRow.jsx

```diff
@@ -183,6 +183,7 @@ class AccountRow extends React.PureComponent {
           [styles['AccountRow--withOwners']]: shouldShowOwners
         })}
         onClick={onClick}
+        data-testid={`balance.account-row.${account._id}`}
```

Comment:
This kind of selector will be hard to maintain, specially when we'll introduce few add  / remove actions. `account._id` will change.

Reply:
_id in fixtures are fixed. We could also fix ids of docs that we add/remove. Or get the testid from the row that was just added in the UI (after adding a doc). I am not so sure about the difficulty here. Do you have a suggestion here ?

Reply:
Pour le moment non et c'est pas bloquant car ton test est pour le moment "statique" ou provient de ta fixture. Mais je sais juste que j'évite de mettre des ID dans le nom des sélecteurs, car dans ma tête après on a un scénario pour créer un account (sans se baser sur des fixtures) et pouvoir se balader dedans. Et donc sélectionner quand on aura un `id` dans le sélecteur ça va être compliqué car pas prédictible. Alors que si tu mets par exemple le nom de l'account, tu le connais vraiment par avance.

Reply:
OK interesting, basing the testid on the name could be a solution indeed :) Thanks.